### PR TITLE
MF-88 - expose TCP port in coap adapter

### LINKED ISSUE #88: Helm charts  doesn't expose CoAP adapter service over TCP port
CoAP adapter should have both UDP and TCP port opened (on the same port number) same as in docker-compose:
https://github.com/mainflux/mainflux/blob/master/docker/docker-compose.yml#L287-L288

CoAP adapter has HTTP listener for `/version` endpoint so opening TCP port is needed. Then that endpoint could probably be used for the liveness probe.

Related issue on core: https://github.com/mainflux/mainflux/issues/1459

### PR DESCRIPTION
**Pull request by blokovi**
Signed-off-by: Ivan Milosevic <iva@blokovi.com>

Resolves #88

### CODE REVIEW

**Review by manuio — DISMISSED** [review]
LGTM

**Review by manuio — DISMISSED** [review]
LGTM

**Review by dusanb94 — DISMISSED** [review]

**Review by dusanb94 — APPROVED** [review]

**Review by manuio — APPROVED** [review]

File: charts/mainflux/templates/adapter_coap-service.yaml [hunk 1]
```diff
@@ -12,3 +12,7 @@ spec:
   ports:
     - port: {{ .Values.adapter_coap.udpPort }}
       protocol: UDP
+      name: "udp-{{ .Values.adapter_coap.udpPort }}"
+    - port: {{ .Values.adapter_coap.udptcpPort }}
+      protocol: TCP
+      name: "tcp-{{ .Values.adapter_coap.udpPort }}"
```
Comment (drasko) [inline charts/mainflux/templates/adapter_coap-service.yaml:18]: How come that we append something called `udpPort` to a `tcp-` prefix? Looks confising.

**Review by drasko — CHANGES_REQUESTED** [review]

File: charts/mainflux/templates/adapter_coap-service.yaml (hunk 1 above)
Comment (blokovi) [reply charts/mainflux/templates/adapter_coap-service.yaml:18]: It's two different ports, items in the list of ports starts with `-`

Comment (drasko) [reply charts/mainflux/templates/adapter_coap-service.yaml:18]: Why is then variable is called `udpPort`?

Comment (blokovi) [reply charts/mainflux/templates/adapter_coap-service.yaml:18]: Here is PR:
https://github.com/mainflux/devops/pull/99

**Pull request merged**